coorx/.github/workflows/deploy.yml

```diff
@@ -0,0 +1,97 @@
+name: Publish Python 🐍 distribution 📦 to PyPI
```

Comment:
why does this show up as an entirely new file? oh, poo; it's in the wrong directory.